diagrams/Makefile

```
@@ -0,0 +1,9 @@
+.PHONY: compile
+compile:
+	mkdir -p ./tmp
+	for f in *.mmd; do NAME=$${f:0:-4}; ./mmdc -i $$f -o $$NAME.svg; convert $$NAME.svg ./tmp/$$NAME.pdf; done
+
+.PHONY: watch
+watch-%: compile
+	okular ./tmp/$(*).pdf &
```

```suggestion
	xdg-open ./tmp/$(*).pdf &
```
Not everyone uses Okular as their preferred PDF viewer. By relying on `xdg-open` we leave it to the user's configuration to pick the PDF viewer of choice.

---

yes, thanks! fixed.